user/pwd.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * @Date: 2024-09-13
+ * @LastEditors: GoKo-Son626
+ * @LastEditTime: 2024-09-13
+ * @FilePath: /caffeinix/user/pwd.c
+ * @Description: Words are cheap so I do.
+ */
+#include "user.h"
+
+#define MAXPATH                 128
+
+int main(int argc, char *argv[]) {
+        char cwd[MAXPATH];
+
+        if (getcwd(cwd, MAXPATH)) {
+                printf("%s", "pwd:getcwd failed\n");
```

Comment:
Just return -1 don't output anything

Reply:
I‘ve been fixed it #27

Reply:
I‘ve been fixed it eb97f8c71e85c0b1daccd31128259d1a39c65a2a

Reply:
Addressed some indentation issues 282cc742e2ce4e7261a587c74cffbe4d95d43504